include/nccl-headers/nvidia/tuner.h

```diff
@@ -0,0 +1,73 @@
+/*************************************************************************
+ * Copyright (c) 2023, NVIDIA CORPORATION. All rights reserved.
+ * Copyright (c) 2023, Meta Platforms, Inc. and affiliates.
+ *
+ * See LICENSE.txt for license information
+ ************************************************************************/
+
+#ifndef NCCL_TUNER_H_
+#define NCCL_TUNER_H_
+
+#include "nccl-headers/net.h"
```

Comment:
I really want this to be a copy directly from upstream NCCL.  That isn't going to work well with the current header, since that has `#include "nccl.h"` right here.  But let's be careful about changes to this file.  I'd be happier if we had a `include/nccl-header/tuner.h` that included dependencies and then this file, similar to what we did with the `net.h` and `error.h` hearders.  We don't have to change this now, but we should do this at some point.